### DISCUSSION TITLE
Deploying to the Web - Talk

### DISCUSSION THREAD

**Discussion by joelpittet** [deployphp/deployer#3142] [General]
I thought I'd share this, I did a presentation at a local Vancouver Drupal User Group that talks about deployment of which this Deployer was the "main event"

https://www.youtube.com/watch?v=bjWxlzDdCiw

Not offended if you don't watch it 😜 just wanted to share my enjoyment of deployer ❤️ Thank you @antonmedv

**Comment by antonmedv** [comment]
Awesome! Thanks for sharing!
